### DISCUSSION TITLE
Examples seem broken in the docs ?

### DISCUSSION THREAD

**Discussion by Twiggeh** [nandorojo/moti#268] [General]
I was trying to check out some of the examples, but they don't appear to work :(
![20230227_15h56m19s_grim](https://user-images.githubusercontent.com/23149166/221582532-d2802404-3415-4d19-a3c4-9d073496b47a.png)

**Comment by nandorojo** [comment]
echo snack is broken for moti unfortunately

**Reply by nandorojo** [reply]
This is fixed now
